projects/igniteui-angular/src/lib/grid/grid.component.ts

```diff
@@ -2411,10 +2411,18 @@ export class IgxGridComponent implements OnInit, OnDestroy, AfterContentInit, Af
     /**
      * @hidden
      */
-    protected _moveColumns(from: IgxColumnComponent, to: IgxColumnComponent) {
+    protected _moveColumns(from: IgxColumnComponent, to: IgxColumnComponent, pos: DropPosition) {
         const list = this.columnList.toArray();
         const fi = list.indexOf(from);
-        const ti = list.indexOf(to);
+        let ti = list.indexOf(to);
```

Comment:
I would use a little more descriptive names for these parameters.
I assume this is to index, if so maybe something like `toIdx` and `fromIdx`